examples/next/faustwp-getting-started/package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "dev": "faustwp dev",
     "build": "faustwp build",
+    "generate": "faustwp generate",
```

Comment:
I wonder if we should call this `generatePossibleTypes`, to reserve `generate` in case we ever need it.